extra_data/export.py

```diff
@@ -242,7 +254,24 @@ def serve_files(path, port, source_glob='*', key_glob='*', **kwargs):
     streamer.start()
     for tid, train_data in data.trains():
         if train_data:
-            streamer.feed(train_data)
+            if data_like == 'file':
+                streamer.feed(train_data)
+            elif data_like == 'online':
+                stacked = stack_detector_data(train_data, 'image.data')
+                merged_data = {}
+                # the data key pretends this is online data from SPB
+                merged_data['SPB_DET_AGIPD1M-1/CAL/APPEND_CORRECTED'] = {
```

Comment:
This should probably select the appropriate source names for raw & corrected data. Is it also meant to handle other detectors (DSSC, LPD, JUNGFRAU)?

Reply:
Pls. see my comment below. At the moment the use case is very "OnDA-centric".